TF_Settings_Web/tsconfig.json

```diff
@@ -16,7 +16,16 @@
         "noEmit": true,
         "jsx": "react-jsx",
         "baseUrl": "src",
-        "typeRoots": ["node_modules/@types", "src/Videos/vidoes"]
+        "typeRoots": ["node_modules/@types", "src/Videos/vidoes"],
```

Comment:
videos looks like an existing typo but can it be solved here?

Reply:
Removed as seems to be un-needed